src/main/java/frc/robot/subsystems/arm/commands/ArmJoystickControl.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+
+public class ArmJoystickControl extends CommandBase {
+    private final Arm arm;
+    private final Joystick shoulderJoystick;
+    private final Joystick elbowJoystick;
+
+    public ArmJoystickControl(Joystick shoulderJoystick, Joystick elbowJoystick) {
+        this.arm = Arm.getInstance();
+        this.shoulderJoystick = shoulderJoystick;
+        this.elbowJoystick = elbowJoystick;
+        addRequirements(arm);
+    }
+
+    @Override
+    public void execute() {
+        arm.setShoulderJointPower(-shoulderJoystick.getY());
+        arm.setElbowJointPower(-elbowJoystick.getY());
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        arm.stop();
```

Comment:
This will make the arm drop in a non-graceful way. Is this the desired behavior?